arxiv/tests/test_license.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for arXiv taxonomy module."""
+from unittest import TestCase
+from arxiv.license import LICENSES, CURRENT_LICENSE_URIS
+
+
+class TestLicense(TestCase):
+    """Tests for the arXiv license definitions."""
+
+    def test_current_license_uris(self):
```

Comment:
This is basically a regression test? Will it need to be updated when the ordering changes? 

Might consider some other validation checks here, for future changes to the licenses. E.g. assert that all items have uris with truthy str values, assert that all items have descriptions, etc. Thinking about things that should be universally true of licenses in this module, and for which there is a non-negligible probability of bungling in the future.

Reply:
This is indeed a regression test (and port of an old test) and would have to be updated with ordering changes. It probably creates more annoyance than actual value, so I'd be fine with removing it.

I've added some validation checks.